### ISSUE TITLE
check_kpts is not called when `FleurBaseWorkChain` is directly submitted

### ISSUE THREAD

**Issue by janssenhenning** [JuDFTteam/aiida-fleur#129]
When the `FleurBaseWorkChain` is called directly with these options the resources are not optimized

```
options = {'resources' : {"num_machines": 5 "num_mpiprocs_per_machine" : 4, "num_cores_per_mpiproc": 12},
           'queue_name' : 'c18m',
           'withmpi' : True,
           'max_wallclock_seconds' : 7200}
```
While for the `FleurSCFWorkChain` the resources are optimized. The reason for this seems to be different defaults for the `optimize_resources` entry (`False` for Base `True` for SCF). Is there a reason for this discrepancy or can this be corrected?

I think calling check_kpts everytime should be the default behaviour

**Comment by Tseplyaev** [comment]
If I remember correctly, there were some complaints about `optimize_resources` and because of that Jens implemented the switch. I would say there is no reason set it to `False` by default - in my opinion it is quite a rare situation when one does not want to use this

**Comment by janssenhenning** [comment]
Okay then I will change the default to `True` for the Base workchain

**Comment by janssenhenning** [comment]
Closed with 614c1c1
